libmisc/ulimit.c

```diff
@@ -11,16 +11,7 @@
 
 #ident "$Id$"
 
-#if HAVE_ULIMIT_H
-# include <ulimit.h>
-# ifndef UL_SETFSIZE
-#  ifdef UL_SFILLIM
-#   define UL_SETFSIZE UL_SFILLIM
-#  else
-#   define UL_SETFSIZE 2
-#  endif
-# endif
-#elif HAVE_SYS_RESOURCE_H
+#if HAVE_SYS_RESOURCE_H
 # include <sys/time.h>		/* for struct timeval on sunos4 */
 /* XXX - is the above ok or should it be <time.h> on ultrix? */
```

Comment:
While moving `sys/time.h` out of this condition you could also remove this comment.

Reply:
I removed them in a separate commit, to keep the changes in this one minimal, for easier review.